### DISCUSSION TITLE
Não salva final do vídeo

### DISCUSSION THREAD

**Discussion by ricardo-bruno** [aclap-dev/video-downloadhelper#253] [Help and Q&A]
### Your question

Estou baixando vídeos da hotmart, mas os últimos segundos não vem junto com o vídeo.

O que fazer?

### Addon & CoApp version and details

Video DownloadHelper
Versão 8.2.1.1 (stable)
Localização do Navegador: pt-BR
Construção de produção
Construído em 2024-02-19
Opções de Construção: browser=chrome, linuxlic=false, noyt=true, target=google

**Comment by paulrouget** [comment]
Known problem. We are working on a fix.
